parm/model_configure

```diff
@@ -1,23 +1,26 @@
-total_member:            1
-PE_MEMBER01:             {{ PE_MEMBER01 }}
 start_year:              {{ start_year }}
 start_month:             {{ start_month }}
 start_day:               {{ start_day }}
 start_hour:              {{ start_hour }}
 start_minute:            0
 start_second:            0
 nhours_fcst:             {{ nhours_fcst }}
+fhrot:                   {{ fhrot }}
 RUN_CONTINUE:            .false.
 ENS_SPS:                 .false.
 dt_atmos:                {{ dt_atmos }}
 calendar:                'julian'
 memuse_verbose:          .false.
-atmos_nthreads:          {{ atmos_nthreads }}
 restart_interval:        {{ restart_interval }}
 output_1st_tstep_rst:    .false.
 write_dopost:            {{ write_dopost }}
 ideflate:                0
 nbits:                   0
+ichunk2d:                -1
+jchunk2d:                -1
+ichunk3d:                -1
+jchunk3d:                -1
+kchunk3d:                -1
```

Comment:
Are these changes appropriate for all configurations of SRW?

Reply:
It should work with the hash of the ufs weather model in SRW. All the regional regression tests in the ufs weather model have these parameters in 'model_configure'. Which configuration do you think will have an issue with them?

Reply:
I have no idea. I don't know what they do, I just want to make sure that they are generic enough not to cause problems with other configurations.

Reply:
I made this change based on the 'model_configure' files of the regression tests in the corresponding hash of the ufs weather model. I think they won't make any problems with any configurations.